src/patterns/service-unavailable-pages/no-replacement-service/index.njk

```diff
@@ -0,0 +1,23 @@
+---
+layout: layout-example.njk
+---
+
+<div class="govuk-width-container">
+
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-xl">Service unavailable</h1>
+        <p class="govuk-body-l">The submit an employment intermediary report service has closed.</p>     
+	      <p class="govuk-body">Contact us if you need to speak to someone about the service and your reports.</p>
+        <p class="govuk-body govuk-!-mb-r0">Telephone:</p>
+        <p class="govuk-body govuk-!-w-bold">03000 555 995</p>
```

Comment:
As per #291, suggest using a reserved number from https://www.ofcom.org.uk/phones-telecoms-and-internet/information-for-industry/numbering/numbers-for-drama here.

Reply:
Good idea. Use 0808 157 3900.